### PR TITLE
Update the styles for new version

### LINKED ISSUE #185: Fix the style for the new version.
- [x] Navigation bar
     - [x] background color
     - [x] text color
     - [x] padding
     - [x] icon color, size, and spacing
 - [x] images
     - [x] padding space and color
 - [x] breadcrumbs position 
 - [x] overall width of layout
 - [ ] table color and spacing
 - [x] scroll bar
 - [x] logo position
 - [x] On this page color and position

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
WIP  to Fix #185

### CODE REVIEW

**Comment by Revathyvenugopal162** [comment]
Live updates are at https://sphinxdocs.ansys.com/dev/index.html.

**Comment by RobPasMue** [comment]
image

It's looking good but... is the scrollbar always going to appear even if it's unnecessary?

**Comment by Revathyvenugopal162** [comment]
> image
> 
> It's looking good but... is the scrollbar always going to appear even if it's unnecessary?

Oh yeah, i did it, once i implemented. i will change it now. Thanks for pointing it Robert.

**Comment by MaxJPRey** [comment]
For dark theme, could it be possible to increase the contrast for the release version?
image

**Comment by RobPasMue** [comment]
> For the dark theme, could it be possible to increase the contrast for the release version? image
> 
> The text could be darker.

Agreed yeah, we should increase the contrast

**Comment by Revathyvenugopal162** [comment]
> For the dark theme, could it be possible to increase the contrast for the release version? image
> 
> The text could be darker.

Done . https://sphinxdocs.ansys.com/dev/getting_started/index.html

**Comment by RobPasMue** [comment]
Is it just my feeling or is the GH button waaaay too small?
image

**Comment by RobPasMue** [comment]
Also:

image

This is atypical issue I face everywhere... would it be possible to get rid of that extra blank space formatted as a console statement, and add it to the right?

**Comment by MaxJPRey** [comment]
Yes @RobPasMue,  GitHub button should be bigger to catch user's attention.

File: .github/workflows/ci_cd.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@ jobs:
 
   doc-deploy-development:
     name: Doc dev version deploy
-    if: github.event_name == 'push'
+    #if: github.event_name == 'push'
```
Comment (jorgepiloto) [inline .github/workflows/ci_cd.yml:52]: Adding this comment so we do not forget about restoring this line.

**Review by RobPasMue — APPROVED** [review]
It's looking much better now, thanks @Revathyvenugopal162

**Comment by Revathyvenugopal162** [comment]
Thank you @MaxJPRey , @RobPasMue and @jorgepiloto for the suggestions and review, Merging and Will solve in another PR, if any modification required.

**Pull request merged**